usr/sbin/p11sak/p11sak.c

```diff
@@ -2561,13 +2589,19 @@ int main(int argc, char *argv[])
 
     /* Prompt for PIN if not already set via option */
     if (!pin) {
-        printf("Please enter user PIN:");
+        printf("Please enter user PIN: ");
```

Comment:
Since `get_pin` requires an `expect` script (due to two-three missing `isatty` calls), this might break some scripts.  Are we sure we want to do that?

Reply:
Hm, good point!  Our tetscases for p11sak don't use expect, but pass the pin as argument via --pin. 
But still there might be other scripts that would be affected.

Opinions? 

Its just 'unusual' to have no space after the colon.....

Reply:
I completely agree with you.  That is why I asked the question.  I think the change should be okay since most `expect` users probably anyway only use something like `"Please enter user PIN:"` without anchoring and then it would still match.

Reply:
So do we want to go with my change? 

If so, then I think I have addressed all review findings so far.

Reply:
I think we want this change.  The `get_pin` function is anyway something we have in multiple variants and should one day consolidate.